.eslintrc

```diff
@@ -1,8 +1,9 @@
 {
   "extends": [
-    "@terrestris/eslint-config-typescript"
+    "@terrestris/eslint-config-typescript",
+    "plugin:react-hooks/recommended"
```

Comment:
We can also make use of the [eslint-config-typescript-react](https://github.com/terrestris/eslint-config-typescript-react) here (instead).

Reply:
I suggest doing that in a followup PR